lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -138,10 +143,15 @@ def draw(self, renderer):
         super().draw(renderer)
         self._children = self._children[:orig_children_len]
 
-    def cla(self):
+    def clear(self):
+        # docstring inherited
         for ax in self.parasites:
-            ax.cla()
-        super().cla()
+            ax.clear()
+        super().clear()
+
+    def cla(self):
+        """Clear the current axes."""
+        self.clear()
```

Comment:
```suggestion
```

Reply:
Removing the `def cla(self)` because `HostAxes` also inherits from Axes with the class factory, as seen on lines 239-242 of `lib/mpl_toolkits/axes_grid1/parasite_axes.py`. 
Note: line numbers specified are before removal of the function.